test/inductor/test_torchinductor_opinfo.py

```diff
@@ -103,6 +103,7 @@ def process(device_type):
 inductor_skips["cpu"] = {
     "linalg.ldl_solve": {b8, f16, f32, f64, i32, i64},  # segfault
     "linalg.lu_solve": {b8, f16, f32, f64, i32, i64},  # segfault
+    "reciprocal": {b8, i32, i64},  # segfault
```

Comment:
This should have been resolved. Merge error?

Reply:
Yes, probably :)